Cargo.toml

```diff
@@ -43,19 +43,19 @@ tracing-subscriber = { version = "0.3.18", features = ["fmt"] }
 
 # kona
 # Switch back to main `kona` once the SingleChainHostCli is merged. https://github.com/op-rs/kona/issues/982
-kona-mpt = { git = "https://github.com/succinctlabs/kona", rev = "b91da21b" }
```

Comment:
Can you link to the branch that you're using for the patch here? We should upstream all of the changes into `kona`.

Reply:
https://github.com/succinctlabs/kona/tree/taehoon/fix-kona-type-annotation

It's this branch on top of ratan/fix-kona-host-compilation.